### PR TITLE
Home page 1.0

### PR DESCRIPTION
**Pull request by typhlosion64**
## Summary of Changes
- i created 2 divs with grid
- one for quotes the other for buttons and stuff

## Imagery
None

### CODE REVIEW

**Review by GCUCoding — DISMISSED** [review]
- Please make sure that all pictures are placed in the preexisting "images" folder, not a new pics folder.
- Please remove patrick.jpg. Unfortunately now that the repo is public we need to maintain a professional image *(I know I know I'm a buzzkill)*

File: js/home.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+var cyber = "Cyber Security";
+var webdev = "Web Development";
+var embedded = "Embedded Development";
+var hardware = "Hardware";
+
+var p0 = "Cyber Security is doing things.";
+var p1 = "Web Dev is doing things.";
+var p2 = "Embedded Dev is doing something.";
+var p3 = "Hardware is doing something.";
```
Comment (elijaholmos) [inline js/home.js:24]:
```suggestion
const cyber = "Cyber Security";
const webdev = "Web Development";
const embedded = "Embedded Development";
const hardware = "Hardware";

const p0 = "Cyber Security is doing things.";
const p1 = "Web Dev is doing things.";
const p2 = "Embedded Dev is doing something.";
const p3 = "Hardware is doing something.";
```

File: js/home.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+var i = 0;
+var txt =
+  "Welcome to the GCU Technology Club's Official Web Site!"; /* The text */
+var speed = 40; /* The speed/duration of the effect in milliseconds */
```
Comment (elijaholmos) [inline js/home.js:4]:
```suggestion
let i = 0;
const txt =
  "Welcome to the GCU Technology Club's Official Web Site!"; /* The text */
const speed = 40; /* The speed/duration of the effect in milliseconds */
```

File: js/home.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+var webdev = "Web Development";
+var embedded = "Embedded Development";
+var hardware = "Hardware";
+
+var p0 = "Cyber Security is doing things.";
+var p1 = "Web Dev is doing things.";
+var p2 = "Embedded Dev is doing something.";
+var p3 = "Hardware is doing something.";
+// code that handles the turning of pages
+var y = 0;
```
Comment (elijaholmos) [inline js/home.js:26]:
```suggestion
let  y = 0;
```

**Review by elijaholmos — CHANGES_REQUESTED** [review]
JS changes; prefer `let` and `const` over `var`
otherwise LGTM

**Review by elijaholmos — APPROVED** [review]
nice job with the changes!

**Pull request merged**